owlbot.py

```diff
@@ -18,7 +18,24 @@
 
 for library in s.get_staging_dirs():
     # put any special-case replacements here
+
     s.move(library)
 
 s.remove_staging_dirs()
 java.common_templates()
+
+s.replace(
+    '**/pom.xml',
+    r'<maven.compiler.target>1.8</maven.compiler.target>',
+    r'<maven.compiler.target>11</maven.compiler.target>'
+)
+s.replace(
+    '**/pom.xml',
+    r'<maven.compiler.source>1.8</maven.compiler.source>',
+    r'<maven.compiler.source>11</maven.compiler.source>'
+)
+s.replace(
+    '.kokoro/**/samples.cfg',
+    r'gcr.io/cloud-devrel-kokoro-resources/java8',
+    r'gcr.io/cloud-devrel-kokoro-resources/java11'
+)
```

Comment:
I've never seen this in other repositories OwlBot configuration file. While this is a python script, you want to use it as a configuration file for OwlBot.